fenix/app/onboarding.fml.yaml

```diff
@@ -9,6 +9,10 @@ features:
         description: if true, juno onboarding is shown to the user.
         type: Boolean
         default: false
+      show-sync:
```

Comment:
I'm wondering if instead of having this field, we add an 'enabled' field for each card - I think that would give more options to experimenter instead of having a hardcoded relationship between sync and widget cards - wdyt?